lib/cast.js

```diff
@@ -378,26 +378,18 @@ function getStrictQuery(queryOptions, schemaUserProvidedOptions, schemaOptions,
   if ('strictQuery' in queryOptions) {
     return queryOptions.strictQuery;
   }
-  if ('strict' in queryOptions) {
```

Comment:
why is this removed? is it meant so that `strict` does not affect `strictQuery` anymore?

Reply:
Yes exactly. We made `strict` tied to `strictQuery` as a workaround for the `strictQuery: true` by default issue. Now that should no longer be necessary.